R/importNGWMN_wml2.R

```diff
@@ -7,7 +7,7 @@
 #' @param asDateTime logical, if \code{TRUE} returns date and time as POSIXct, if \code{FALSE}, character
 #' @param tz character to set timezone attribute of datetime. Default is an empty quote, which converts the 
 #' datetimes to UTC (properly accounting for daylight savings times based on the data's provided time zone offset).
-#' Possible values to provide are "America/New_York","America/Chicago", "America/Denver","America/Los_Angeles",
+#' Possible values are "America/New_York","America/Chicago", "America/Denver","America/Los_Angeles",
```

Comment:
It is...but in the documentation, I want to emphasize that these are the timezones that the US will almost most certainly have. That's why I changed it from "to provide" to "are".